Lib/test/test_print.py

```diff
@@ -155,40 +155,49 @@ def test_string_with_excessive_whitespace(self):
 
         self.assertIn('print("Hello World", end=" ")', str(context.exception))
 
-    def test_print_string_with_stream_redirection(self):
+    def test_stream_redirection_hint_for_py2_migration(self):
         import sys
```

Comment:
There's no need for this to be a function level import - it can just go at the top of the test module.